staging/src/k8s.io/client-go/kubernetes/typed/admissionregistration/v1/mutatingwebhookconfiguration.go

```diff
@@ -80,6 +82,16 @@ func (c *mutatingWebhookConfigurations) Get(ctx context.Context, name string, op
 
 // List takes label and field selectors, and returns the list of MutatingWebhookConfigurations that match those selectors.
 func (c *mutatingWebhookConfigurations) List(ctx context.Context, opts metav1.ListOptions) (result *v1.MutatingWebhookConfigurationList, err error) {
+	if watchListOptions, hasWatchListOptionsPrepared, watchListOptionsErr := watchlist.PrepareWatchListOptionsFromListOptions(opts); watchListOptionsErr != nil {
+		return &v1.MutatingWebhookConfigurationList{}, watchListOptionsErr
```

Comment:
Seems like this error should fall back to a normal list.

Reply:
updated the code to fall back in that case too.